<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectClass</class>
 <widget class="QMainWindow" name="ProjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>571</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_title1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>21</width>
        <height>121</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>单变量预测</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>指数平滑法</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>季节指数法</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ARMA法</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>pushButton_3</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>pushButton</zorder>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_title2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>21</width>
        <height>121</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>因素预测</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>211</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>BP神经网络</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>小波神经网络</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>GARCA模型</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_title3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>21</width>
        <height>121</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>组合预测法</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>BP+指数平滑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>小波+季节指数平滑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>GARCA+指数平滑</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="Area2" name="area2" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="Area1" name="area1" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="QSplitter" name="splitter2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>144</width>
       <height>15</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>预测二个月</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label2_data">
      <property name="text">
       <string>300</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter1">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>31</y>
       <width>144</width>
       <height>15</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>预测一个月</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label1_data">
      <property name="text">
       <string>50</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>144</width>
       <height>15</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>预测三个月</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label3_data">
      <property name="text">
       <string>400</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Area3" name="area3" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="Area4" name="area4" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>290</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1145</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>单变量预测</string>
    </property>
    <addaction name="action11"/>
    <addaction name="action12"/>
    <addaction name="action13"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>因素预测</string>
    </property>
    <addaction name="action21"/>
    <addaction name="action22"/>
    <addaction name="action23"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>组合预测法</string>
    </property>
    <addaction name="action31"/>
    <addaction name="action32"/>
    <addaction name="action33"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action11">
   <property name="text">
    <string>指数平滑法</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>季节指数法</string>
   </property>
  </action>
  <action name="action13">
   <property name="text">
    <string>ARMA法</string>
   </property>
  </action>
  <action name="action21">
   <property name="text">
    <string>BP神经网络</string>
   </property>
  </action>
  <action name="action22">
   <property name="text">
    <string>小波神经网络</string>
   </property>
  </action>
  <action name="action23">
   <property name="text">
    <string>GARCA模型</string>
   </property>
  </action>
  <action name="action31">
   <property name="text">
    <string>BP+指数平滑</string>
   </property>
  </action>
  <action name="action32">
   <property name="text">
    <string>小波+季节指数平滑</string>
   </property>
  </action>
  <action name="action33">
   <property name="text">
    <string>GARCA + 指数平滑</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Area2</class>
   <extends>QWidget</extends>
   <header>area2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Area1</class>
   <extends>QWidget</extends>
   <header>area1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Area3</class>
   <extends>QWidget</extends>
   <header>area3.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Area4</class>
   <extends>QWidget</extends>
   <header>area4.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="project.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>openBox()</slot>
 </slots>
</ui>
